src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -44,11 +45,23 @@ private static string GetBuildIntegratedProjectCacheFilePath(RestoreRequest requ
             return null;
         }
 
+        /// <summary>
+        /// Combines <paramref name="cacheRoot"/> + "project.nuget.cache"
+        /// </summary>
+        /// <param name="cacheRoot">Path to cache root</param>
+        /// <param name="projectPath">Not used</param>
+        /// <returns>The combined path</returns>
+        [Obsolete("Use overload withou projectPath argument")]
         public static string GetProjectCacheFilePath(string cacheRoot, string projectPath)
```

Comment:
Similar feedback as before.

I don't think removing a warning for an unused param is a good enough reason to introduce more long term work by adding more APIs.

Reply:
Removed